src/lib/util/FileHandler.ts

```diff
@@ -1,96 +1,49 @@
-import fs = require('fs');
-import {Stats} from 'fs';
+import {Stats, promises as fs, constants as fsConstants} from 'fs';
```

Comment:
Changed this entire file to use the `fs.promises` syntax instead of the callback-based version. This is safe to do now, since that syntax has been generally available since node v14, and the current LTS version is 16.